### DISCUSSION TITLE
The problems on data usage and GIS

### DISCUSSION THREAD

**Discussion by jason-muljadi** [albertkun/21S-ASIAAM-191A#34] [Past Thinking Caps]
The fundamental of data collection is always going to be from a human effort, whether or not it is automated. This makes data unreliable to begin with. The way that data is collected relies on the biases of the collectors, but it also relies on another factor: the honesty of the information collected.

Without regulation of data collection, there may be answers to questions that are not completely accurate. Yet, regulations also affect the process itself, as more regulations impede greater turnout of collection. There is always a seesaw between quantity and quality with data sets, and data is also collected based on the same interest of individuals. Specifically for GIS, the agenda of certain folks may be aligned to redlining a district or to engage in gerrymandering so as to obtain a political advantage. It could also be to elevate a community in terms of what they need but it can be also used to erase a community's existence through geographic means. Ignorance also breeds problematic datasets and GIS projects, as the data presented to the general public may not have encapsulated a population's true thoughts and desires. A lack of GIS or other data-mapping tool experience can also limit the effectiveness of transmitting crucial information on a large-scale setting to allow help and empowerment in the community to occur.

No one data set is perfect, which is why it is important to think about each data set critically. 

![image](https://user-images.githubusercontent.com/81675803/113526617-97770400-956f-11eb-9d09-c0f70635c3aa.png)
![image](https://user-images.githubusercontent.com/81675803/113526645-ad84c480-956f-11eb-953e-571e0bd4a23c.png)
![image](https://user-images.githubusercontent.com/81675803/113526660-c3928500-956f-11eb-8ab0-3292a81f1476.png)
![image](https://user-images.githubusercontent.com/81675803/113526680-d6a55500-956f-11eb-8bec-03dcc91d2ce1.png)

The example below shows the district line of the City of Industry, courtesy of Google Maps and Emily Pham
![image](https://user-images.githubusercontent.com/81675803/113526840-60edb900-9570-11eb-81b1-14579f069c99.png)

**Comment by epham36** [comment]
> "A lack of GIS or other data-mapping tool experience can also limit the effectiveness of transmitting crucial information on a large-scale setting to allow help and empowerment in the community to occur."

This is so true! It is so unfortunate that the way data/data visualization are presented can pose such a risk to effective community change. At a time when we see personal politics, economic interests, and scientific research butting heads so frequently, effective data visualization not only promotes equitable institutional action but also helps uplift historically marginalized voices. This can especially be helpful in governmental/institutional proceedings which are often purposefully made opaque and inaccessible to the general public. Effective, communicative, and accessible visualizations of data can highlight the narratives of those who often go unheard. 

When big data/big tech/those who perform research feel so inaccessible, experience with visualization tools like GIS and other software are a way for the community itself to gain agency over the stories they tell and the interventions they advocate for.

**Comment by brillianz** [comment]
Indeed, it is important to think about our data sets critically. Within the realm of research, the question of objectivity is more and more being interrogated to instead opt towards worldviews with greater interpretations and implications for communities. More recently, this has also been featured with quantitative research methods, as [Dr. Velez et al. feature in this 2017 article](https://www.tandfonline.com/doi/full/10.1080/13613324.2017.1377675). Their contributions towards a linkage of CRT (critical race theory) and quant methods introduce an exposure to methods otherwise often considered "foreign" to our communities. The emergence of quantitative methods should not be forgotten by community-centered researchers and organizers, for it is these same units of analysis that our neighborhoods will ultimately be evaluated through. Through a critical paradigm, we can work towards implementing this Critical Quant further into multiple fields, whether social research methods, Ethnic Studies, or statistics.
